shared/src/question-generators/asymptotics/between.ts

```diff
@@ -262,12 +261,20 @@ export function generateBaseFunction(variant: string, random: Random): ProductTe
           maxDenominator: 3,
           random,
         }),
-        logexponent: sampleFraction({
-          fractionProbability: 0,
-          minInt: -17,
-          maxInt: 17,
-          random,
-        }),
+        logexponent: random.choice([
```

Comment:
Could be shortened to sampleFraction({
          fractionProbability: 0,
          minInt: 1,
          maxInt: 17,
          random,
        }).mul(random.choice([-1,1]))